### DISCUSSION TITLE
PWA Support

### DISCUSSION THREAD

**Discussion by Silther** [Floorp-Projects/Floorp#492] [Feature Request]
Please add support for PWAs or normal Web Apps like YouTube

**Comment by creeper-0910** [comment]
Why did you post with the discussion closed?

**Comment by Ryosuke-Asano** [comment]
I closed. There is a same discussion
https://github.com/Floorp-Projects/Floorp/discussions/411

**Reply by Silther** [reply]
Sorry, looked only in the issue section for feature requests

**Reply by Ryosuke-Asano** [reply]
Do not worry.

**Comment by Ryosuke-Asano** [comment]
https://github.com/Floorp-Projects/Floorp/discussions/411#discussioncomment-7800301
